package.json

```diff
@@ -11,6 +11,7 @@
     "react": "^16.13.0",
     "react-bootstrap": "^1.0.0-beta.17",
     "react-dom": "^16.13.0",
+    "react-icons": "^3.9.0",
```

Comment:
Może po prostu coś z fontawesome? Po co nam kolejne dependency.

Reply:
aaa bo to bylo w tym poprzednim mozna to usunac

Reply:
`npm uninstall react-icons` jak coś.